remote-routing.js

```diff
@@ -8,6 +8,26 @@ module.exports = function(Model, options) {
   });
 };
 
+var remoteMethod = Model.remoteMethod;
+
+Model.remoteMethod = function(name, config) {
+  var disable;
+
+  remoteMethod.call(Model, name, config);
+
+  if (options.only && options.only.length) {
```

Comment:
options is the mixin options that gets passed though to check against, config is the remoteMethod config 

eg. 

Model.remoteMethod('setup', { 'accepts': [] })